src/olympia/addons/forms.py

```diff
@@ -296,9 +296,9 @@ def __init__(self, *args, **kw):
             if not acl.action_allowed(self.request, 'Addons', 'Edit'):
                 form.disabled = (app and self.addon.is_featured(app))
 
-    def save(self):
+    def save(self, addon=None):
```

Comment:
I'm confused. Why would you need to pass `addon` to the `save()` method when this form already takes an `addon` at init ?